teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', parents_permissiom: true)
+    super(age, name, parents_permissiom)
```

Comment:
- [ ] Kindly assign `parents_permissiom` properly in the `#super` method, since it uses the colon notation, to avoid argument errors.

  ```
  super(age, name, parents_permissiom: parents_permissiom)
  ```